src/Routes/LoginRoute.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
```

Comment:
I'd really like this to be in src/Routes/LoginRoute/LoginRoute.js

It may just be my OCD, but I'd like to see each Route to live in its own directory. Let me know how you feel about that.

Thanks